Hazelcast.Test/Hazelcast.Client.Test.Serialization/JsonValueSerializationTest.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Hazelcast.IO.Serialization;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Hazelcast.Client.Test.Serialization
+{
+    [TestFixture]
+    public class JsonValueSerializationTest
```

Comment:
Missing category

Reply:
`HazelcastJsonValueSerializationTest` is a better name I guess.

Reply:
Renamed